subscriber/podaac_data_downloader.py

```diff
@@ -284,6 +284,14 @@ def run(args=None):
     logging.info("Downloaded Files: " + str(success_cnt))
     logging.info("Failed Files:     " + str(failure_cnt))
     logging.info("Skipped Files:    " + str(skip_cnt))
+
+    #create citation file if success > 0
+    if success_cnt > 0:
+        try:
+            pa.create_citation_file(short_name, provider, data_path, token, args.verbose)
+        except:
+            logging.debug("Error generating citation")
```

Comment:
```suggestion
        except:
            logging.debug("Error generating citation", exc_info=True)
```
IMO we should never ignore stack traces.

Reply:
agreed, i added this in- the difficulty is if a non-PO.DAAC collection is used and it doesn't supply all the bits, i don't want to break the rest of subscriber for a this feature.

Reply:
i added the exc_info=True to the warning message